### DISCUSSION TITLE
想问有没有推荐的使用L7的开源项目？

### DISCUSSION THREAD

**Discussion by wsqstar** [antvis/L7#2282] [Q&A]
发现GitHub上有7k左右的项目引用了L7 但是很难搜索到相关的项目到底是使用L7的。
因为这可能是一个使用or **入门L7（对我来说）** 的重要参考。
GitHub上的数据如下
https://github.com/antvis/L7/network/dependents
![image](https://github.com/antvis/L7/assets/24500520/141d48f9-7b5e-42d8-acee-e3d9615a9f4e)

**Comment by lzxue** [comment]
还是看官网案例吧
